.github/workflows/release.yaml

```
@@ -71,14 +71,23 @@ jobs:
         run: mv release ../
 
       - name: Checkout helm chart
+        if: ${{ github.event.inputs.step == '5' || github.event.inputs.step == '5-publish-helm' }}
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           persist-credentials: false
           repository: cilium/charts
           path: "./cilium-charts"
 
-      - name: Move helm charts to upper directory
-        run: mv cilium-charts ../charts
+      - name: Set up charts repository git configuration
+        if: ${{ github.event.inputs.step == '5' || github.event.inputs.step == '5-publish-helm' }}
```

From the backports it looks like the first conditional here is unnecessary (same feedback for the other cases below)
```suggestion
        if: ${{ github.event.inputs.step == '5-publish-helm' }}
```